### PR TITLE
Ports/psoc6 nwdocs

### PR DESCRIPTION
**Pull request by IFX-Anusha**
Documentation for network module & completion monitor updated with network details

### CODE REVIEW

**Review by jaenrig-ifx — APPROVED** [review]
Nice :)

File: docs/psoc6/quickref.rst [hunk 1]
```diff
@@ -236,3 +236,77 @@ See :ref:`machine.RTC <machine.RTC>` ::
...
+
+Scan for the available wireless networks using 
+
+::
+
+   wlan.scan()             
+    
+Scan function returns a list of tuple information about access points
+(ssid, bssid, channel, RSSI, security, hidden) .There are 7 levels of security.
+
```
Comment (boramonideep) [inline docs/psoc6/quickref.rst:261]: Suggestion: 
This part could be highlighted to draw attention. 
image

**Review by boramonideep — APPROVED** [review]
One suggestion, else looks great. Thanks @IFX-Anusha

File: docs/psoc6/quickref.rst (hunk 1 above)
Comment (IFX-Anusha) [reply docs/psoc6/quickref.rst]: Thanks, @boramonideep! Done

**Review by NikhitaR-IFX — APPROVED** [review]

**Pull request merged**